libcontainer/system/linux.go

```diff
@@ -35,7 +35,16 @@ func Execv(cmd string, args []string, env []string) error {
 		return err
 	}
 
-	return unix.Exec(name, args, env)
+	return Exec(name, args, env)
+}
+
+func Exec(cmd string, args []string, env []string) error {
+	err := unix.Exec(cmd, args, env)
```

Comment:
Doesn't this line need to be inside the loop? As written if we get EINTR we will loop forever.

Reply:
oop

Reply:
🤦‍♂️ thanks.